book/IMaEA.md

```diff
@@ -5380,7 +5380,7 @@ The system test suite
 
 positional arguments:
   dir_or_file           root directory under which the testsuite searches for tests or asingle file which tests are run (defaults to the config
-                        file'slocation)
+                        file's location)
```

Comment:
Not sure if this text corresponds with anything elsewhere. I don't recall making a similar change.